planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/data_structs.hpp

```diff
@@ -262,25 +262,28 @@ struct Info
   }
 };
 
-template <typename Object>
-struct LanesObjects
+struct TargetLaneLeadingObjects
 {
-  Object current_lane{};
-  Object target_lane_leading{};
-  Object target_lane_trailing{};
-  Object other_lane{};
-
-  LanesObjects() = default;
-  LanesObjects(
-    Object current_lane, Object target_lane_leading, Object target_lane_trailing, Object other_lane)
-  : current_lane(std::move(current_lane)),
-    target_lane_leading(std::move(target_lane_leading)),
-    target_lane_trailing(std::move(target_lane_trailing)),
-    other_lane(std::move(other_lane))
+  ExtendedPredictedObjects moving;
+  ExtendedPredictedObjects stopped;
+
+  // for objects outside of target lanes, but close to its boundaries
+  ExtendedPredictedObjects stopped_outside_boundary;
```

Comment:
I prefer `stopped_at_bound`

Reply:
Done in https://github.com/autowarefoundation/autoware.universe/pull/9372/commits/816193b9d46fa56b568afc544bf0bf15b81afeb4